### PR TITLE
Validation: Added Missing Descriptions

### PR DESCRIPTION
**Pull request by dereklh7**
Added missing metadata description, fixed broken links, and added column headers.

### CODE REVIEW

File: BingMaps/spatial-data-services/geocode-dataflow-api/get-status-of-a-geocode-job.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Get Status of a Geocode Job | Microsoft Docs"
+description: "Describes how to get the status of a geocode job b and provides supported HTTP methods, the URL template, template parameters, and examples."
```
Comment (brittmsantos) [inline BingMaps/spatial-data-services/geocode-dataflow-api/get-status-of-a-geocode-job.md:3]: "geocode job b"--seems like the b at the end is unnecessary?

**Comment by dereklh7** [comment]
@stevemunk @rbrundritt

Checking in on the status of this PR to see if there are any changes that need to be implemented for this PR to be approved and merged. Thanks!

**Comment by stevemunk** [comment]
@dereklh7 Looking now...

File: BingMaps/spatial-data-services/geocode-dataflow-api/index.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Geocode Dataflow API | Microsoft Docs"
+description: "Describes the Geocode Dataflow API and provides a table that outlines a description for various resources relating to the API."
```
Comment (stevemunk) [inline BingMaps/spatial-data-services/geocode-dataflow-api/index.md:3]:
```suggestion
description: "The overview page for the Geocode Dataflow API, which uses REST URLs to geocode and reverse-geocode large sets of spatial data, with information on what you need to do in order to use these API along with links to articles that describe the data schema, how to create and check on jobs, download results, sample input/output files, sample code and a walkthrough."
```

File: BingMaps/spatial-data-services/index.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Bing Spatial Data Services | Microsoft Docs"
+description: "Describes Bing Spatial Data Services and provides a table that outlines a description for various resources relating to Bing Spatial Data Services."
```
Comment (stevemunk) [inline BingMaps/spatial-data-services/index.md:3]:
```suggestion
description: "The overview page for the Spatial Data Services section, which uses REST URLS to geocode, store and query spatial data. Describes what you can do with these API and contains a list of links to topics in this section with descriptions for each."
```

File: BingMaps/spatial-data-services/public-data-sources/fourthcoffeesample.md [hunk 4]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "FourthCoffeeSample | Microsoft Docs"
+description: "Describes the FourthCoffeeSample data source and provides a table that outlines the data type and example value for various properties."
```
Comment (stevemunk) [inline BingMaps/spatial-data-services/public-data-sources/fourthcoffeesample.md:3]:
```suggestion
description: "Describes the FourthCoffeeSample data source, a sample data source used when learning how to query and get data source information using the Bing Spatial Data Services, with a table that outlines the data type and example value for each of its properties."
```

File: BingMaps/spatial-data-services/public-data-sources/index.md [hunk 5]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Public Data Sources | Microsoft Docs"
+description: "Describes Bing Spatial Data Services' public data sources and provides a table that outlines a description for various data sources."
```
Comment (stevemunk) [inline BingMaps/spatial-data-services/public-data-sources/index.md:3]:
```suggestion
description: "The overview page for the Public Data Sources section, with a list of links to each data source available and a description for each."
```

File: BingMaps/spatial-data-services/query-api/index.md [hunk 6]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Query API | Microsoft Docs"
+description: "Describes the Query API and provides a table that outlines a description for various resources related to the Query API."
```
Comment (stevemunk) [inline BingMaps/spatial-data-services/query-api/index.md:3]:
```suggestion
description: "The overview page for the Query API section, with links to each of the articles in this section and a description for each."
```

File: BingMaps/spatial-data-services/query-api/query-api-sample-code-csharp.md [hunk 7]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Query API Sample Code (C#) | Microsoft Docs"
+description: "Provides C# sample code for the Query API. This sample code provides an example of how to query a data source using the Query API."
```
Comment (stevemunk) [inline BingMaps/spatial-data-services/query-api/query-api-sample-code-csharp.md:3]:
```suggestion
description: "The Query API Sample Code, written in C#, provides examples of how to query a data source using the Query API."
```

**Review by stevemunk — CHANGES_REQUESTED** [review]

**Comment by dereklh7** [comment]
@stevemunk I've implemented the requested changes. Please let me know if there are any other additional issues preventing approval and merge for this PR. Thanks! Hope you have a great rest of your day!

**Comment by stevemunk** [comment]
@dereklh7 can you please remove the file: BingMaps/spatial-data-services/public-data-sources/navteqeu.md? It is outdated. I removed it via a different PR, which created a merge conflict that cannot be resolved using the github website, thought it might just be easier that fixing the merge conflict using the console, not something I've needed to do.

**Comment by dereklh7** [comment]
@stevemunk In an attempt to resolve this merge conflict in VS Code, I ended up reverting all changes originally made to navteqeu.md - that file now appears unedited from before this PR. Does that work? If not, please let me know.

**Comment by stevemunk** [comment]
@dereklh7 Since that file is no longer in main, it seems that if you merged/sync'd with main, it should remove it from your working branch as well. It shouldn't be showing up at all as a part of this PR, I'm worried that if I merge, it will create this file again in main.

**Pull request merged**